### DISCUSSION TITLE
enforcedHostRegexp referenced incorrectly in Helm Values.yaml file

### DISCUSSION THREAD

**Discussion by jaguer0** [linkerd/linkerd2#11973] [General]
The values.yaml file notes [enforcedHostRegexp](https://github.com/linkerd/linkerd2/blob/34ef27307a34c3849daa90cabfdff24cd676de5e/viz/charts/linkerd-viz/values.yaml#L359C3-L359C21) as it's own Key which is incorrect , this needs to be `dashboard.enforcedHostRegexp` like it says in the [README.md ](https://github.com/linkerd/linkerd2/tree/main/viz/charts/linkerd-viz) 

My first attempt was as noted in values.yaml and it lead to an expected error being thrown due to my attempt of accessing the viz dashboard when enforced host is not set. Updating it to the way in the README and the [web.yaml](https://github.com/linkerd/linkerd2/blob/34ef27307a34c3849daa90cabfdff24cd676de5e/viz/charts/linkerd-viz/templates/web.yaml#L103) file allowed it to work correctly. 

Hoping to save someone some time

**Comment by alpeb** [comment]
> The values.yaml file notes [enforcedHostRegexp](https://github.com/linkerd/linkerd2/blob/34ef27307a34c3849daa90cabfdff24cd676de5e/viz/charts/linkerd-viz/values.yaml#L359C3-L359C21) as it's own Key which is incorrect

Not sure I understand your point. `enforcedHostRegexp` is located under the `dashboard` entry in `values.yaml`, so it's referred to as `dashboard.enforcedHostRegexp`.
